src/views/Login.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="login-form">
+    <v-content>
+      <v-container fluid>
+        <v-row align="center" justify="center">
+          <v-col cols="12" sm="8" md="4">
+            <v-card elevation="12">
```

Comment:
seems to be the same vue code here as before. Would a shared component / partial work here?